static/js/timerender.js

```diff
@@ -317,11 +317,18 @@ export function get_full_datetime(time) {
     };
 }
 
+function set_data_tippy_content(message, time_elem) {
+    time_elem.attr("data-tippy-content", message.full_date_str + " " + message.full_time_str);
+}
```

Comment:
Can you post a screenshot of what the Tippy tooltip looks like?  We may also want to play with a 2-line format, which wasn't possible with `title` but would likely look nicer here.

Reply:
Posted while opening PR. This is how it looks with the changes:
![image](https://user-images.githubusercontent.com/44665669/122441743-7b560100-cfbb-11eb-8c07-bd57e8848657.png)

Will try doing a double line format and post a screenshot soon.

Reply:
2-line format:
Do you want us to try one of these?
![image](https://user-images.githubusercontent.com/44665669/122457057-e0195780-cfcb-11eb-96b1-bad7368881e3.png)
![image](https://user-images.githubusercontent.com/44665669/122457220-0e973280-cfcc-11eb-9a9d-83920872a1ea.png)
(The second looks good to my eyes :)

Reply:
I think maybe we just want to use a nicer date format, like:
```
Saturday, June 3 at {time} # if current year
Saturday, June 3, 2019 at {time} # if another yet
```

And then the other thing I notice is that we probably want the popover centered when the right sidebar is present, which is a TippyJS configuration thing @amanagr may need to help with.

Reply:
Do we want to avoid timezone specific `(UTC+5.5)` part?

Reply:
Sorry for the confusion: I think we want the timezone indication, which confirms for you the current timezone being used for messages in Zulip.  @alya thoughts on where to put it?  I can imagine spelling it out as e.g. 

`United States/Los Angeles (UTC +5.5)`  on a second line, if we want to be maximally unambiguous.

Reply:
Okay. Showing full timezone string to keep it unambiguous makes sense to me. There's also another question of whether we want to show the full month or just an abbreviation to keep the content short. I asked the same in zulip at [frontend>timestamp title disappears on opening message link](https://chat.zulip.org/#narrow/stream/6-frontend/topic/timestamp.20title.20disappears.20on.20opening.20message.20link/near/1214524) thinking we can discuss everything there. Feels like a full month name is better.

I'll do some iterations on this with full month string and complete timezone string in different line orders and post there for us to have a better picture to decide what to go with :slightly_smiling_face:

Reply:
Yeah, I think it's a good idea to show the timezone abbreviation to keep it short. If we are going with the abbreviation, we'll need the `(UTC -7)` part to keep timezones like `IST` less ambiguous.